stampit.js

```diff
@@ -126,9 +125,9 @@ stampit = function stampit(options) {
     addStatic(fixed, options.static);
   }
 
-  var factory = function Factory(properties, args) {
-    properties = properties ? mixer.merge({}, fixed.props, properties) : deepClone(fixed.props);
-    var instance = mixer.mixin(create(fixed.methods), fixed.refs, properties); // props are taking over refs
+  var factory = function Factory(refs, args) {
+    var instance = mixer.mixin(create(fixed.methods), fixed.refs, refs);
+    mixer.mergeUnique(instance, fixed.props); // props are safely merged into refs
```

Comment:
Disadvantage: any ref object might receive new properties from the `fixed.props`.

Advantages: `stamp()` argument now serves both purposes - refs and props.
```js
var stamp = stampit.props({ url: { host: 'localhost' } });
var instance = stamp({ url: { port: 8080 } }); // deeply merged -> { url: { host: 'localhost', port: 8080 } }

var stamp = stampit.refs({ url: { host: 'localhost', port: 8080 } });
instance = stamp.refs({ method: 'GET' }); // shallwo mixed -> { method: 'GET', url: { host: 'localhost', port: 8080 } }
```

See my point? It's a win-win situation.